### PR TITLE
 PAYARA-3240 Cleanup sonar warnings for nucleus/cluster/admin - Payara4

### PR DESCRIPTION
**Pull request by Cousjava**

### CODE REVIEW

File: nucleus/cluster/admin/src/main/java/com/sun/enterprise/v3/admin/cluster/ClusterCommandHelper.java [hunk 1]
```diff
@@ -79,11 +79,11 @@
  */
```
Comment (MeroRai) [inline nucleus/cluster/admin/src/main/java/com/sun/enterprise/v3/admin/cluster/ClusterCommandHelper.java]: Needs portions copyright header

File: nucleus/cluster/admin/src/main/java/com/sun/enterprise/v3/admin/cluster/CreateInstanceCommand.java [hunk 2]
```diff
@@ -88,7 +88,8 @@
         description="Create Instance")
 })
 public class CreateInstanceCommand implements AdminCommand {
```
Comment (MeroRai) [inline nucleus/cluster/admin/src/main/java/com/sun/enterprise/v3/admin/cluster/CreateInstanceCommand.java]: Needs portions copyright header

File: nucleus/cluster/admin/src/main/java/com/sun/enterprise/v3/admin/cluster/ListClustersCommand.java [hunk 3]
```diff
@@ -103,6 +102,7 @@
     @Inject
     private Clusters allClusters;
 
+    @Override
```
Comment (MeroRai) [inline nucleus/cluster/admin/src/main/java/com/sun/enterprise/v3/admin/cluster/ListClustersCommand.java]: Needs portions copyright header

File: nucleus/cluster/admin/src/main/java/com/sun/enterprise/v3/admin/cluster/ListInstancesCommand.java [hunk 4]
```diff
@@ -162,7 +162,7 @@ public void execute(AdminCommandContext context) {
     }
 
     private void noStatus(List<Server> serverList) {
-        if (serverList.size() < 1) {
+        if (serverList.isEmpty()) {
```
Comment (MeroRai) [inline nucleus/cluster/admin/src/main/java/com/sun/enterprise/v3/admin/cluster/ListInstancesCommand.java]: Needs portions copyright header

File: nucleus/cluster/admin/src/main/java/com/sun/enterprise/v3/admin/cluster/StopClusterCommand.java [hunk 5]
```diff
@@ -123,7 +123,6 @@ public void execute(AdminCommandContext context) {
             logger.warning(msg);
             report.setActionExitCode(ExitCode.FAILURE);
             report.setMessage(msg);
-            return;
```
Comment (MeroRai) [inline nucleus/cluster/admin/src/main/java/com/sun/enterprise/v3/admin/cluster/StopClusterCommand.java]: Needs portions copyright header

**Comment by Cousjava** [comment]
Jenkins test please

**Comment by Pandrex247** [comment]
Cancelled the build, more copyright is required.

**Comment by MeroRai** [comment]
Jenkins test please

**Pull request merged**